server/Customer.java

```diff
@@ -4,6 +4,8 @@
 
 public class Customer {
 
+	private boolean accountLocked = false;
```

Comment:
Assigning `false` here is not needed, as the default value for a `boolean` is already `false`. All types have their defaults, for example, default values for objects (created from classes) is `null`. 

If you had an object of `Customer` type, this is the same:

```java
Customer customer = null;
Customer customer;
```